config/kcp/kustomization.yaml

```diff
@@ -1,10 +1,21 @@
----
```

Comment:
yamllint is going to complain about this and the other whitespace changes.

Reply:
The entire project is not checked for yamllint. There are a lot of changes in multiple files that needs to be fixed. I will raise a separate PR for that.